app/renderer/components/tabs/tab.js

```diff
@@ -323,7 +336,10 @@ class Tab extends React.Component {
           ref={(node) => { this.tabSentinel = node }}
           className={css(styles.tab__sentinel)}
         />
-        <div className={css(styles.tabId)}>
+        <div className={css(
+          styles.tab__identity,
+          this.props.centralizeTabIcons && styles.tab__content_centered
+          )}>
```

Comment:
would you please delete two whitespaces before the brackets?

Reply:
wow weird standard accepted this, nice catch

Reply:
Confirmed this was fixed on https://github.com/brave/browser-laptop/pull/10691/files#diff-4f164dab3a38a1443d7ba3288fef641eR342

On the commit: https://github.com/brave/browser-laptop/commit/94ba1a1557d4646f7e36b69c0f110f59f3b835aa#diff-4f164dab3a38a1443d7ba3288fef641eR331